FindOnPage.md

```diff
@@ -0,0 +1,664 @@
+# WebView2Find API
+
+## Background
+
+The WebView2Find API offers methods and events for text finding and navigation
+within a WebView2 control. It enables developers to programmatically initiate find
+operations, navigate find results, suppress default UI, and customize find configurations
+like query and find direction. It also tracks the status of operations, indicating
+completion, match count changes, and match index changes.
+
+## Examples
+
+
+#### Description
+
+To initiate a find operation in a WebView2 control, use the `StartFind` method.
+This method allows setting the find term and find parameters via the
+`ICoreWebView2FindConfiguration` interface. Only one find session can be active per
+webview environment. Starting another with the same configuration will adjust
+the active match index based on the selected Find Direction.
+### Create/Specify a Find Configuration
+#### WIN32 C++
+
+```cpp
+
+//! [InitializeFindConfiguration]
+wil::com_ptr<ICoreWebView2FindConfiguration> AppWindow::InitializeFindConfiguration(const std::wstring& findTerm)
+{
+    // Query for the ICoreWebView2Environment5 interface.
+    auto webView2Environment5 = m_webViewEnvironment.try_query<ICoreWebView2Environment5>();
+    CHECK_FEATURE_RETURN(webView2Environment5);
```

Comment:
`CHECK_FEATURE_RETURN_EMPTY`? (I can't find definitions of these macros, but it seems that the caller expects null to be returned on failure.)

Reply:
Please validate. Follow up with me if there's something in WIL or something more standard we can use instead throughout our sample app code.